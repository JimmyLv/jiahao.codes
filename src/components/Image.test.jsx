import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Image from './Image';

test('Link changes the class when hovered', () => {
  const wrapper = shallow(<Image />);
  expect(wrapper).toMatchSnapshot();
});
