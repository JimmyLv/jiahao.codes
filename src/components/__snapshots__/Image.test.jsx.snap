// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
@media only screen and (min-width:740px) {
  .c0 {
    width: 80vw;
    left: 50%;
    -webkit-transform: translateX(-50%);
    -ms-transform: translateX(-50%);
    transform: translateX(-50%);
    min-width: 740px;
  }
}

@media only screen and (max-width:740px) {
  .c0 {
    margin-left: -1rem;
    margin-right: -1rem;
  }
}

<Image
  alt=""
  className="c0"
  fadeIn={true}
/>
`;
